libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/Skills/SkillHttpClient.cs

```diff
@@ -25,8 +25,39 @@ public SkillHttpClient(HttpClient httpClient, ICredentialProvider credentialProv
             _conversationIdFactory = conversationIdFactory;
         }
 
+        /// <summary>
+        /// Uses the SkillConversationIdFactory to create or retrieve a Skill Conversation Id, and sends the activity.
+        /// </summary>
+        /// <param name="originatingAudience">The oauth audience scope, used during token retrieval. (Either https://api.botframework.com or bot app id.)</param>
+        /// <param name="fromBotId">The MicrosoftAppId of the bot sending the activity.</param>
+        /// <param name="toSkill">The skill to create the conversation Id for.</param>
+        /// <param name="callbackUrl">The callback Url for the skill host.</param>
+        /// <param name="activity">The activity to send.</param>
+        /// <param name="cancellationToken">A cancellation token.</param>
+        /// <returns>Async task with invokeResponse.</returns>
+        public async Task<InvokeResponse> PostActivityAsync(string originatingAudience, string fromBotId, BotFrameworkSkill toSkill, Uri callbackUrl, Activity activity, CancellationToken cancellationToken)
```

Comment:
I suppose you could pass in the `ClaimsIdentity` to get `originatingAudience` and `fromBotId`... Not feedback I think should be applied for R8, but perhaps a minor cleanup for R9/R10.